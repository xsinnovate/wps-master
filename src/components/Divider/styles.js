import { StyleSheet } from 'react-native';

export default StyleSheet.create({
  container: {
    paddingVertical: 15,
  },
  line: {
    height: 1,
    backgroundColor: '#dfdfdf',
  },
});
