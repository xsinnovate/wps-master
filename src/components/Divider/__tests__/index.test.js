import React from 'react';
import renderer from 'react-test-renderer';
import Divider from '..';

describe('<Divider />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Divider />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
