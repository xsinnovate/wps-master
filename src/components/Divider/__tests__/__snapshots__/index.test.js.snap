// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> renders correctly 1`] = `
<View
  style={
    Object {
      "paddingVertical": 15,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#dfdfdf",
        "height": 1,
      }
    }
  />
</View>
`;
